utils/create_scap_delta_tailoring.py

```diff
@@ -178,7 +179,7 @@ def create_tailoring(args):
 
     tailoring_root = ET.Element('xccdf-1.2:Tailoring')
     version = ET.SubElement(tailoring_root, 'xccdf-1.2:version',
-                            attrib={'time': datetime.datetime.utcnow().isoformat()})
+                            attrib={'time': datetime.datetime.now(datetime.UTC).isoformat()})
```

Comment:
From the CI results we see that the new way doesn't work with Python 2.7 and 3.6 which we still support. The function hasn't been removed from 3.12, it's only deprecated. So I see 2 options: 1. don't do this change now, 2. create a wrapper function that would call either `now(datetime.UTC)` or `utcnow` depending on Python version or their presence.